docs/guides/access-control.mdx

```diff
@@ -0,0 +1,256 @@
+---
+sidebar_position: 5
+title: Access Control
+description: Complete guide to authentication, access control and token management in ReductStore
+---
+
+import CodeBlock from "@theme/CodeBlock";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<head>
+    <link rel="canonical" href="https://www.reduct.store/docs/guides/access-management"/>
+</head>
+
+# Access Control
+
+## Concepts
+
+ReductStore provides straightforward access management based on token authentication and permissions.
+
+### Token Authentication
+
+ReductStore uses token-based authentication to secure the API. A client must send a valid token in the `Authorization` header in order to access the API.
+A ReductStore instance must have an initial token to enable authentication and access management. The initial token is configured using the `RS_API_TOKEN` environment variable and has full access permissions.
+
+Access tokens are immutable and can't be updated. If you need to change the token, you must create a new token and update the clients with the new token. The value of the token is a random string that is generated when the token is created and is only available at that time.
+
+:::info
+If you don't need access control, you can disable token authentication by leaving `RS_API_TOKEN` unset.
+:::
+
+### Permissions and Access Control
+
+Each token has a set of permissions that define what actions can be performed with the token:
+
+- `full_access`: Allows the token to perform any action on the API.
+- `read`: List of buckets allowed to read data from entries.
```

Comment:
Can you clarify the set of permissions? I didn't get the`read` and `write` when reading quickly. Maybe something like:

- full_access: Allows the token to perform any action on the API without any restrictions.

- read: Allows the token to read data from specific buckets. The list of buckets that can be accessed for reading is defined when the token is created.

- write: Allows the token to write data to specific buckets, which includes adding new entries or deleting existing ones. The list of buckets that can be accessed for writing is defined when the token is created.